spec/blockchain/encoding.md

```diff
@@ -314,13 +265,11 @@ Because Tendermint only uses a Simple Merkle Tree, application developers are ex
 
 ## JSON
```

Comment:
do we want to pull this out into its own page where we can better define the spec of the json encodeer?

Reply:
Probably. Note that the JSON encoding is not consistent: some components use the native Golang encoder, others use the Amino/Protobuf-style encoder. At some point we should just pick one and stick to it.